### PR TITLE
RavenDB-15215: Reduce the response size when loading the documents page

### PR DESCRIPTION
**Pull request by garayx**

### CODE REVIEW

File: src/Raven.Server/Documents/DocumentsStorage.cs [hunk 1]
```diff
@@ -1366,20 +1367,48 @@ public void AssertMetadataKey(string id, BlittableJsonReaderObject document, Doc
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        private static Document ParseDocument(JsonOperationContext context, ref TableValueReader tvr, DocumentFields fields)
+        private static Document ParseDocument(JsonOperationContext context, ref TableValueReader tvr, DocumentFields fields, bool skipMetadataData = false)
         {
             if (fields == DocumentFields.All)
             {
+                var id = TableValueToId(context, (int)DocumentsTable.Id, ref tvr);
```
Comment (grisha-kotler) [inline src/Raven.Server/Documents/DocumentsStorage.cs]: Can we do this in only in `StudioCollectionsHandler.cs` and not here?

**Review by grisha-kotler — CHANGES_REQUESTED** [review]

File: src/Raven.Server/Documents/DocumentsStorage.cs (hunk 1 above)
Comment (garayx) [reply src/Raven.Server/Documents/DocumentsStorage.cs]: done

File: src/Raven.Server/Web/Studio/StudioCollectionsHandler.cs [hunk 2]
```diff
@@ -120,6 +129,31 @@ public Task PreviewCollection()
...
+                {
+                    [Constants.Documents.Metadata.Key] = metadata
+                };
+
+                metadata.Modifications.Remove(Constants.Documents.Metadata.Counters);
+                metadata.Modifications.Remove(Constants.Documents.Metadata.Attachments);
+                metadata.Modifications.Remove(Constants.Documents.Metadata.TimeSeries);
+            }
+
+            if (document.Data.Modifications != null)
```
Comment (grisha-kotler) [inline src/Raven.Server/Web/Studio/StudioCollectionsHandler.cs]: Can this be done in `WriteDocument`?

Comment (garayx) [reply src/Raven.Server/Web/Studio/StudioCollectionsHandler.cs]: done

**Review by grisha-kotler — APPROVED** [review]

**Review by ml054 — APPROVED** [review]

**Pull request merged**